maintainers/maintainer-list.nix

```diff
@@ -1448,6 +1448,11 @@
     githubId = 5149377;
     name = "Amine Chikhaoui";
   };
+  amiryal = {
+    name = "Amir Yalon";
+    email = "nixpkgs-maintainer@please.nospammail.net";
```

Comment:
Email is optional if you have `githubId` set

Reply:
No, not a fake. I have a premium Fastmail account which allows me to have the `please` alias, and I can swap `anything@alias.domain.example` for `alias@domain.example`.